include/deal.II/multigrid/mg_transfer_global_coarsening.templates.h

```diff
@@ -1695,7 +1904,7 @@ namespace internal
             [&](const auto &, const auto &, const auto c) {
               for (unsigned int i = 0;
                    i < transfer.schemes[1].n_dofs_per_cell_coarse;
-                   i++)
+                   ++i)
                 weights_1[cell_local_chilren_indices[c][i]] =
```

Comment:
Not related to this pr, but here is a typo (and somewhere at the place of declaration of course)
```suggestion
                weights_1[cell_local_children_indices[c][i]] =
```